CDCgov/NEDSS-Modernization
# [Fix] Cucumber tests

## Description

The Cucumber tests were not running for Page Builder after a recent change.  ([#648](https://github.com/CDCgov/NEDSS-Modernization/pull/648)).  There were also some failing tests that made it through while Cucumber was not running.  These tests were fixed except the Page Rule tests.

## Steps to verify

1.  Run all test by calling `./gradlew clean test`
2. Ensure that the Cucumber Report exists for Page Builder
3. Ensure that the Cucumber Report exists for Modernization API
4. Weep
